src/fsharp/FSComp.txt

```diff
@@ -73,6 +73,7 @@ buildProductNameCommunity,"F# Compiler for F# %s"
 pickleErrorReadingWritingMetadata,"Error reading/writing metadata for the F# compiled DLL '%s'. Was the DLL compiled with an earlier version of the F# compiler? (error: '%s')."
 245,tastTypeOrModuleNotConcrete,"The type/module '%s' is not a concrete module or type"
 tastTypeHasAssemblyCodeRepresentation,"The type '%s' has an inline assembly code representation"
+246,optsUnrecognizedLanguageVersion,"Unrecognized value '%s' for --langversion use --langversion:? for complete list"
```

Comment:
Change to:

> Unrecognized value '%s' for --langversion. Please use --langversion:? for complete list

Reply:
Note that `optsUnrecognizedLanguageVersion` is only for the command-line switch, we need a different message for the LangVersion property since we can't tell people to use `--lang version:?` in a project file context.

Reply:
agreed

Reply:
Perhaps we can dump the valid values in the error message?

Reply:
This is what C# does though:
````
csc /langversion:78.9

error CS1617: Invalid option '78.9' for /langversion. Use '/langversion:?' to list supported values.
````

This is F#:
````
c:\kevinransom\fsharp>artifacts\bin\fsc\Debug\net472\fsc.exe --langversion:78.2

error FS0246: Unrecognized value '78.2' for --langversion use --langversion:? for complete list

c:\kevinransom\fsharp>
````

Reply:
@KevinRansom, when I type wrong version identifier, having the program telling me "type this to show list of version" is good, but if it can also list the actual valid values as well prentending I just did that, it would be better.